src/traces/parcoords/defaults.js

```diff
@@ -0,0 +1,101 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Lib = require('../../lib');
+var attributes = require('./attributes');
+var hasColorscale = require('../../components/colorscale/has_colorscale');
+var colorscaleDefaults = require('../../components/colorscale/defaults');
+
+// todo unify with scatter/line_defaults.js (which needs to handle line width and dash as well)
+var handleLineDefaults = function lineDefaults(traceIn, traceOut, defaultColor, layout, coerce) {
+    var markerColor = (traceIn.marker || {}).color;
+
+    coerce('line.color', defaultColor);
+
+    if(hasColorscale(traceIn, 'line')) {
+        colorscaleDefaults(traceIn, traceOut, layout, coerce, {prefix: 'line.', cLetter: 'c'});
+    }
+    else {
+        var lineColorDflt = (Array.isArray(markerColor) ? false : markerColor) || defaultColor;
+        coerce('line.color', lineColorDflt);
+    }
+};
+
+
+function dimensionsDefaults(traceIn, traceOut) {
+    var dimensionsIn = traceIn.dimensions || [],
+        dimensionsOut = traceOut.dimensions = [];
+
+    var dimensionIn, dimensionOut;
+
+    function coerce(attr, dflt) {
+        return Lib.coerce(dimensionIn, dimensionOut, attributes.dimensions, attr, dflt);
+    }
+
+    for(var i = 0; i < dimensionsIn.length; i++) {
+        dimensionIn = dimensionsIn[i];
+        dimensionOut = {};
+
+        if(!Lib.isPlainObject(dimensionIn) || !Array.isArray(dimensionIn.values)) {
```

Comment:
@alexcjohnson having caught up on this item with Étienne, the addition of a UI interaction was deferred for now. I created a `visible` attribute in the meantime to have API control.